dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -582,14 +579,13 @@ public void save(Folder folder, User user, boolean respectFrontEndPermissions) t
 
 	}
 
-
+	@CloseDBIfOpened
 	public Folder findSystemFolder() throws DotDataException {
-		return ffac.findSystemFolder();
+		return folderFactory.findSystemFolder();
 	}
 
 
-
-
+	// todo: not sure if this should be a transaction
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)